src/Symfony/Component/Mime/NamedAddress.php

```diff
@@ -11,11 +11,22 @@
 
 namespace Symfony\Component\Mime;
 
+use Symfony\Component\Mime\Exception\InvalidArgumentException;
+
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  */
 final class NamedAddress extends Address
 {
+    /**
+     * Simple regex to match a structure like 'Name <email@address.com>'.
+     * It matches anything between the first < and last > as email address.
+     * This will allow to use a single string to construct a NamedAddress, which can be convenient to use in
+     * config, and allows to have more readable config.
+     * This does not try to cover all edge cases for address.
+     */
+    private const FROM_STRING_PATTERN = '~(?<displayName>[^<]*)<(?<addrSpec>.*)>[^>]*~';
```

Comment:
Not sure we need this pattern as a constant here, it is only used once

Reply:
The constant allows to have the comment and the regex together, so I think we can keep it like this.